scripts/build_clone_docs.sh

```diff
@@ -11,20 +11,31 @@ CUR_DIR="$(cd $(dirname $0) && pwd)"
 
 git_clone_doc_tags() {
 
-    #get all tags from the repo
-    TAGS=$(git tag -l --sort=-v:refname)
-    #get the latest tag from repo
-    LATEST_TAG=$(git tag -l --sort=v:refname | tail -1)    
+    #get all tags we want to clone from the repo
+    TAGS_TO_CLONE=$($CUR_DIR/tagScript.js $(git tag -l)) 
+
+    #get rid of characters we dont need
+    TAGS=${TAGS_TO_CLONE//[[\',\]]/}
+    
+    #sort the array and set the tag with the greatest value as the latest tag
+    LATEST_TAG=($(for each in ${TAGS[@]}; do echo $each; done | sort | tail -1))
+
+    #parse out only the major.minor values for the latest tag
+    LATEST_MAJOR_MINOR=${LATEST_TAG%.*}
 
     git checkout $LATEST_TAG
-    echo -e -n "{\"latest\":\""$LATEST_TAG"\", \"versions\":["  >> docversions.json
+    echo -e -n "{\"latest\":\""$LATEST_MAJOR_MINOR"\", \"versions\":["  >> docversions.json
 
     for TAG in $TAGS; do
-    echo -e -n \"$TAG\", >> temp.json
+
+    #parse out only the major and minor values of each tag
+    TAG_MAJOR_MINOR=${TAG%.*}
+    
+    echo -e -n \"$TAG_MAJOR_MINOR\", >> temp.json
```

Comment:
same thing try to have the script prepare the data the way we need, instead of altering the data in 2 places